apps/web/pages/auth/forgot-password/[id].tsx

```diff
@@ -185,6 +186,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
         },
         id,
         csrfToken: await getCsrfToken({ req: context.req }),
+        ...(await serverSideTranslations(context.locale || "en", ["common", "home"])),
```

Comment:
What is the purpose of home here?
I don't see home.json files.

Also, I have mostly seen and used this approach for Server side translations. Not sure which is the correct or best appraoch
<img width="1737" alt="Screenshot 2023-03-09 at 2 05 17 PM" src="https://user-images.githubusercontent.com/1780212/223965753-86209717-955c-455f-a1f1-9516f062cd07.png">

Reply:
Damn, you're right. Home shouldn't be included, I copy pasted 🥲 You got me there. 
As for the the ssr hydrate — yes that's the most common approach. However, the forgot password page is a static page, meaning it doesn't have access to other server stuff (like prisma), which is why the `ssrInit()` can't be used. 

Gonna push a fix for the translation now.